README.md

```diff
@@ -55,8 +55,8 @@ send them timely push notifications via [FCM (Firebase Cloud Messaging)](https:/
       ```kotlin
       // build.gradle.kts
       dependencies {
-          implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:3.0.3")
-          implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:3.0.3")
+          implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:3.2.0-alpha.2")
```

Comment:
I assume we'll need to bump this to a non-alpha release version before we merge everything in?

Reply:
Yep, I expect to build at least one more alpha SDK/test app, then we'll bump version to official 3.2.0